<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mimetypeviewer</class>
 <widget class="QWidget" name="mimetypeviewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>398</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>395</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mimetypes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="1">
    <widget class="QTreeView" name="mimetypeTreeView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>180</width>
       <height>280</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="expandsOnDoubleClick">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>0</number>
     </attribute>
     <attribute name="headerMinimumSectionSize">
      <number>500</number>
     </attribute>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="headingLabel">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Configure file associations</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="indent">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" rowspan="2">
    <widget class="QFrame" name="mimeFrame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>260</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QGroupBox" name="applicationsGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>260</width>
        <height>140</height>
       </rect>
      </property>
      <property name="title">
       <string>Default application</string>
      </property>
      <widget class="QPushButton" name="chooseApplicationsButton">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>90</y>
         <width>110</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>&amp;Choose...</string>
       </property>
      </widget>
      <widget class="QFrame" name="applicationLabelFrame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>241</width>
         <height>51</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="appIcon">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="applicationLabel">
          <property name="text">
           <string>None</string>
          </property>
          <property name="indent">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="patternsGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>260</width>
        <height>60</height>
       </rect>
      </property>
      <property name="title">
       <string>Patterns</string>
      </property>
      <widget class="QLabel" name="patternsLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>230</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>*.txt *.xml</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="mimetypeGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>260</width>
        <height>90</height>
       </rect>
      </property>
      <property name="title">
       <string>Mimetype</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="iconLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Icon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="searchTermLineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Type mimetype or file ending...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>searchTermLineEdit</tabstop>
  <tabstop>mimetypeTreeView</tabstop>
  <tabstop>chooseApplicationsButton</tabstop>
  <tabstop>dialogButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
